src/contract.rs

```diff
@@ -770,6 +915,11 @@ pub fn query_info(deps: Deps) -> StdResult<InfoResponse> {
     let token1 = TOKEN1.load(deps.storage)?;
     let token2 = TOKEN2.load(deps.storage)?;
     let lp_token_address = LP_TOKEN.load(deps.storage)?;
+    let fees = FEES.load(deps.storage)?;
+
+    let owner = OWNER.load(deps.storage)?;
+    let owner = Some(owner).map(|o| o.unwrap().to_string());
```

Comment:
Pretty sure this will panic if owner is unset the way we have it here.

```suggestion
    let owner = OWNER.load(deps.storage)?.map(|o| o.into_string());
```